plugin.video.cbc/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.cbc"
+       name="Canadian Broadcasting Corp (CBC)"
+       version="4.0.8"
```

Comment:
The Leia version is 4.0.6, which might conflict with future Python 2 updates. I don't know if you guys have a public testing repository, but I would highly recommend adding "+matrix" to it or bumping the version to something like 5.0.0.

Reply:
You should use `4.0.6+matrix.1` for the Matrix version.